packages/terminal-extension/schema/plugin.json

```diff
@@ -81,6 +81,12 @@
       "type": "boolean",
       "default": false
     },
+    "disposeOnExit": {
```

Comment:
Is _dispose_ a good term for the end user? What about `closeOnExit`?

Reply:
`closeOnExit` sounds better, and is more aligned with the already-existing `shutdownOnClose`.